packages/profile-sync-controller/package.json

```diff
@@ -102,7 +102,7 @@
   "dependencies": {
     "@metamask/base-controller": "^8.0.0",
     "@metamask/keyring-api": "^17.2.0",
-    "@metamask/keyring-controller": "^19.2.0",
+    "@metamask/keyring-controller": "^19.2.1",
```

Comment:
Nit: can we add a changelog for this bump

Reply:
Oh yes, good catch :) Let me add this.

Reply:
[1654ce3](https://github.com/MetaMask/core/pull/5373/commits/1654ce315e96249749f8b9bcc9faea005a0627f1)